### ISSUE TITLE
Article 8.  Is "Initial Organization" section needed in bylaws?

### ISSUE THREAD

**Issue by ktchu** [jbardhan/csgf-draftdocs#16]
- Is the initial setup procedure typically included in bylaws?  It feels like they shouldn't be necessary, but I don't have enough experience with bylaws to know.
- This question also applies to article 3.5

**Comment by goxberry** [comment]
Good point! Usually, this sort of thing isn't in the bylaws, since it is a one-time setup. Normally, temporary officers are elected at the first meeting of an organization, along with a committee to draft bylaws. At a second meeting (possibly more, depending on issues with the bylaws that need working out), the bylaws are adopted via a motion from the chairman of the bylaws committee, followed by a majority vote. Then, (more) permanent officers are elected according to the procedure set out in the bylaws. (_Robert's Rules of Order Newly Revised_, 11th ed., pp. 553-561)

**Comment by jbardhan** [comment]
Hi @ktchu @goxberry 

We are drafting an "initial plan" document which will not be a formal part of the bylaws, but instead an informational document for members and potential members.  We believe the final article of the bylaws addresses the formal requirements.  Please let us know if this addresses your concerns.

Thanks very much,
Jay

**Comment by goxberry** [comment]
@jbardhan tl;dr: I don't think this article needs to be in the bylaws. Justification follows.

In my previous comment in this thread, I noted the procedure suggested for in-person assemblies. The situation is a little different here because the nature of the organization demands electronic participation from those interested in attending meetings who cannot be there in person due to travel-related issues (e.g., the CSGF Annual Meeting).

My take is: everything listed in this article has either already been done (you already have a board), or will get done in the next few meetings of the association (everything else). The bylaws are meant to codify rules and concepts that will last the life of the organization; this article doesn't achieve that goal because it's basically useless after adoption of the bylaws. Everything you want to do in this article you can do through other means.
